apps/explorer/src/components/common/BlockExplorerLink/BlockExplorerLink.tsx

```diff
@@ -52,15 +53,15 @@ export const BlockExplorerLink: React.FC<Props> = (props: Props) => {
     return null
   }
 
-  const url = getExplorerUrl(networkId, type, identifier)
+  const url = getBlockExplorerUrl(networkId, type, identifier)
   const label = labelProp || (useUrlAsLabel && url) || abbreviateString(identifier, 6, 4)
 
   return (
     <ExternalLink href={url} target="_blank" rel="noopener noreferrer" className={className}>
       <span>{label}</span>
       {showLogo && (
         <LogoWrapper
-          title={`Open it on ${networkId === Network.GNOSIS_CHAIN ? 'Gnosisscan' : 'Etherscan'}`}
+          title={`Open it on ${CHAIN_INFO[networkId].explorerTitle}`}
```

Comment:
ubernit: Suince there's these url and label const there, I would think is nicer to restructure the `CHAIN_INFO[networkId]` so the markup is simpler